cli/src/semgrep/commands/ci.py

```diff
@@ -615,8 +615,9 @@ def ci(
                 "  Semgrep Cloud Platform is still processing the results of the scan, they will be available soon:"
             )
 
+        ref_if_deployment_name = f"&ref={metadata.branch}" if metadata.branch else ""
```

Comment:
nit: what does deployment_name have to do with this? Did you mean more like `ref_if_available`?